addons/mail/static/src/core/common/create_poll_dialog.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="mail.CreatePollDialog">
+    <Dialog size="'md'" title.translate="Create a poll">
+        <div class="w-100 mb-3">
+            <label class="w-100">
+                <span class="d-inline-block mb-2 fw-bold">Question</span>
+                <input type="text" name="poll_question" t-model="state.question" class="form-control rounded" placeholder="What question do you want to ask?" t-ref="question"/>
+                <div t-if="questionMissing" class="text-danger form-text ms-1">The question cannot be empty.</div>
+            </label>
+        </div>
+        <div class="mb-3">
+            <span class="mb-2 d-inline-block fw-bold">Answers</span>
+            <CreatePollDialogAnswer t-foreach="state.answers" t-as="answer" t-key="answer_index" model="answer" deletable="state.answers.length > 2" onClickRemove="() => this.onClickRemoveAnswer(answer_index)"/>
+            <button class="btn btn-primary align-self-end" t-on-click="onClickAddAnswer" t-att-disabled="!canAddAnswer"><i class="fa fa-plus"/> Add another answer</button>
+            <div t-if="answersMissing" class="text-danger form-text ms-1">At least two answers are required</div>
+        </div>
+        <label class="d-flex align-items-center mb-3">
+            <span class="fw-bold me-2">Duration</span>
+            <select class="form-select" name="poll_duration" style="width: fit-content;" t-model="state.duration">
+                <option value="1">1 hour</option>
+                <option value="4">4 hours</option>
+                <option value="8">8 hours</option>
+                <option value="24">24 hours</option>
+            </select>
+        </label>
+        <label class="d-flex align-items-center">
+            <span class="fw-bold me-2">Allow multiple answers</span>
+            <input type="checkbox" name="poll_allow_multiple_answers" t-model="state.allowMultipleAnswers"/>
+        </label>
```

Comment:
- looks best for `Add another answer` to be a `btn-link`
- could have "allow multiple answers" and "Duration" inline, so less space is wasted

This would look like this:
<img width="652" height="450" alt="Image" src="https://github.com/user-attachments/assets/a53241dd-dafb-4378-9b3a-03542f8224d1" />